datasets/us/ss_most_wanted/us_ss_most_wanted.yml

```diff
@@ -0,0 +1,15 @@
+title: "United States Secret Service Most Wanted Fugitives"
+entry_point: crawler.py
+prefix: usss-most-wanted
+summary: "Wanted fugitivies listed on United States Secret Service website."
+description: >
+  This dataset publishes the details of the most wanted fugitives listed on the United States Secret Service website.
+url: https://www.secretservice.gov/investigations/mostwanted
+publisher:
```

Comment:
Please add a description for the secret service - usually I find a line or two I can adapt from their about page. Imagine someone from another country reading this who doesn't understand the difference between the secret service, FBI, and CIA. What's the secret service?